jax/_src/numpy/lax_numpy.py

```diff
@@ -436,8 +436,50 @@ def fmax(x1: ArrayLike, x2: ArrayLike) -> Array:
   """
   return where(ufuncs.greater(x1, x2) | ufuncs.isnan(x2), x1, x2)
 
-@util.implements(np.issubdtype)
+
 def issubdtype(arg1: DTypeLike, arg2: DTypeLike) -> bool:
+  """Return True if arg1 is equal or lower than arg2 in the type hierarchy.
+
+  JAX implementation of :func:`numpy.issubdtype`.
+
+  The main difference in JAX's implementation is that it properly handles
```

Comment:
Is this true for NumPy 2.0 as well?

Reply:
It may be different with the new-style dtypes, but `ml_dtypes` doesn't implement that yet.

I put the doctest below so we'll get a CI failure in this docstring if that ever changes.